lib/active_fulfillment/services/shopify_api.rb

```diff
@@ -86,7 +86,7 @@ def send_app_request(action, headers, data)
         end
       end
 
-      log :info, "GET response action=#{action} in #{"%.4fs" % realtime} #{response}"
+      log :info, "GET response action=#{action} in #{format('%.4fs', realtime)} #{response.to_s[0..70]}"
```

Comment:
`ActiveSupport` is a dependency of this gem, and it adds the `String#truncate` method:
https://github.com/rails/rails/blob/4940cc49ddb361d584d51bc3eb4675ff8ece4a2b/activesupport/lib/active_support/core_ext/string/filters.rb#L66

So you can do `response.to_s.truncate(SOME_LENGTH, omission: "[response truncated]")`